### PR TITLE
Measure and expose inbound RTP packet loss

### PR DESCRIPTION
**Pull request by kris-lab**

### CODE REVIEW

File: libjanus_rtpbroadcast.c [hunk 1]
```diff
@@ -3309,6 +3396,9 @@ json_t *cm_rtpbcast_source_to_json(cm_rtpbcast_rtp_source *src, cm_rtpbcast_sess
 	json_object_set_new(s, "max", json_real(src->stats.max));
 	json_object_set_new(s, "cur", json_real(src->stats.cur));
 	json_object_set_new(s, "avg", json_real(src->stats.avg));
+	/* TODO: we average losess across video and audio @landswellsong */
+	json_object_set_new(s, "cur_loss", json_real((src->stats.current_loss[AUDIO]+src->stats.current_loss[VIDEO])/2.0));
```
Comment (kris-lab) [inline libjanus_rtpbroadcast.c]: @njam should we keep track of packet loss separately for Audio and Video - in case of QoS?

Comment (njam) [inline libjanus_rtpbroadcast.c]: I would track it separately, as the two streams are completely separate for a long time.

**Comment by njam** [comment]
How will packet loss be exposed on the stats endpoint? Lost packets per stream per second?

**Comment by kris-lab** [comment]
Copying @landswellsong from https://github.com/cargomedia/janus-gateway-rtpbroadcast/pull/69#issuecomment-202876129
> The stats about the packet loss estimated are now shown in list command. cur_loss is an estimate for the window of last source_avg_time seconds as regular stats, avg_loss is an estimate for the whole time the connection is on.

**Comment by kris-lab** [comment]
Seems to be working for me. Gonna merge this and continue...

**Comment by kris-lab** [comment]
@njam I am merging this as cannot continue on another ones (too many conflicts). Currently we can use "bad connection simulator" to check if logic works and for development&debugging for e.g. checking how UDP packets loss influence REMB at WebRTC.

**Pull request merged**